Content.Server/_DV/CosmicCult/CosmicCultSystem.cs

```diff
@@ -0,0 +1,209 @@
+using Content.Server._DV.CosmicCult.EntitySystems;
+using Content.Server.Actions;
+using Content.Server.AlertLevel;
+using Content.Server.Audio;
+using Content.Server.Chat.Systems;
+using Content.Server.GameTicking.Events;
+using Content.Server.Pinpointer;
+using Content.Server.Popups;
+using Content.Server.Station.Systems;
+using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CosmicCult;
+using Content.Shared.Alert;
+using Content.Shared.Damage;
+using Content.Shared.DoAfter;
```

Comment:
Remove unused using directive
```suggestion
using Content.Shared.Alert;
using Content.Shared.DoAfter;
```